packages/compass-import-export/src/components/import-options/import-options.jsx

```diff
@@ -11,6 +11,10 @@ import styles from './import-options.module.less';
 import createStyler from '../../utils/styler.js';
 const style = createStyler(styles, 'import-options');
 
+const fileInputStyles = css({
+  margin: '5px auto 20px',
```

Comment:
Can we use spacing here?

Reply:
We definitely should. Though I think the issue here is a bit different.

I had thought that a better way than styling these spaces everywhere would be to move `FormFieldContainer` in compass components and use that instead, so all the forms are going to look the same, is a bit too much for this PR but we have a ticket to unify the forms in the same epic.

Since I moved this one from another file and is in the import/export dialog (which in the worse case is going to be rebuilt from scratch) I'd lean to keep it that way for now.

Reply:
Added this to https://jira.mongodb.org/browse/COMPASS-6035